backend/build.gradle

```diff
@@ -55,9 +55,9 @@ dependencies {
 
     // Okta dependencies
     implementation 'com.okta.spring:okta-spring-boot-starter:2.0.0'
-    implementation 'com.okta.sdk:okta-sdk-api:3.0.2'
-    runtimeOnly 'com.okta.sdk:okta-sdk-impl:3.0.2'
-    runtimeOnly 'com.okta.sdk:okta-sdk-httpclient:3.0.2'
+    implementation 'com.okta.sdk:okta-sdk-api:8.2.1'
```

Comment:
a question for the team: do we have a procedure for upgrading dependencies? Given this is a pretty major upgrade, wondering if we have /should introduce some process into ensuring we're not breaking anything inadvertently going from 3.0 --> 8.2

Reply:
I'm going to do a complete regression testing of all of the okta apis to make sure that nothing fails unexpectedly.

Reply:
@fzhao99 no real procedure other than what Boban outlined; we just have to do manual regression testing unfortunately.